### PR TITLE
adding etrraform for ec2

### PR DESCRIPTION
**Pull request by ramanji1990**
Hi @pujithadevops ,

Created terraform code can you review and provide your approval.

Regards,
Ram

### CODE REVIEW

File: ec2-terraform.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_instance" "web" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.micro"
```
Comment (pujithadevops) [inline ec2-terraform.txt]: change instance type to t2.micro

**Review by pujithadevops — CHANGES_REQUESTED** [review]
can you please do the changes

**Comment by ramanji1990** [comment]
Added new changes.

**Review by pujithadevops — APPROVED** [review]
looks good

**Pull request merged**